app-admin/vaultwarden/files/vaultwarden-envfile-1.33.2.patch

```diff
@@ -0,0 +1,63 @@
+As upstream defaults create directories where binary is executed (/usr/bin/), this patch is needed to change paths to Gentoo specific locations
+--- a/.env.template
++++ b/.env.template
+@@ -5,33 +5,29 @@
+ ## Be aware that most of these settings will be overridden if they were changed
+ ## in the admin interface. Those overrides are stored within DATA_FOLDER/config.json .
+ ##
+-## By default, Vaultwarden expects for this file to be named ".env" and located
```

Comment:
I'm thinking, we source this file for our usage, right?  So it's supposed to be true shell.

Reply:
scratch that, we use ENV_FILE